UI/ModdedMapScreen.cs

```diff
@@ -241,7 +271,12 @@ private void Update()
 
         internal void OnMapScreenShow()
         {
-            _peekComponent?.EndPeek();
+            PeekComponent?.EndPeek();
```

Comment:
This should be an explicit null check, unity overrides this and can lead to issues. Same with every other occurrence inside of a monobehavior